EvidenceApi.Tests/V1/E2ETests/EvidenceRequestsTest.cs

```diff
@@ -216,13 +216,13 @@ public async Task ReturnBadRequestWhenServiceAndResidentIdAreEmpty()
             var team = "";
             var residentId = "";
             var uri = new Uri($"api/v1/evidence_requests?team={team}&residentId={residentId}", UriKind.Relative);
-            var response = await Client.GetAsync(uri).ConfigureAwait(true);
+            var response = await Client.GetAsync(uri);
 
             response.StatusCode.Should().Be(400);
         }
 
         [Test]
-        public async Task CanSendANotificationUploadConfirmationToResident()
```

Comment:
Spoke to Bogdan about the wording and he agrees that the Hackney officer should be called 'Staff' in the code (consistent with `StaffSelectedDocumentTypes` etc.